src/View/Helper/GooglePlacesHelper.php

```diff
@@ -0,0 +1,117 @@
+<?php
+namespace Geo\View\Helper;
+
+use Geo\View\Helper\GoogleMapHelper;
+
+/**
+ * This is a CakePHP helper that helps users to integrate Google Places
+ * into their application by only writing PHP code. This helper depends on jQuery.
+ *
+ * CodeAPI: http://code.google.com/intl/de-DE/apis/maps/documentation/javascript/basics.html
+ * Icons/Images: http://gmapicons.googlepages.com/home
+ *
+ * @author Michael Heit
+ * @link http://www.dereuromark.de/2010/12/21/googlemapsv3-cakephp-helper/
+ * @license MIT License (http://www.opensource.org/licenses/mit-license.php)
+ * @property \Cake\View\Helper\HtmlHelper $Html
+ */
+class GooglePlacesHelper extends GoogleMapHelper {
+
+	/**
+	 * Needed helpers
+	 *
+	 * @var array
+	 */
+	public $helpers = [ 'Form', 'Html' ];
+
+
+	/**
+	 * Wrapper function for control like Form->input
+	 *
+	 * @param string $fieldName name of input field
+	 * @param array $fieldOptions associative array of settings are passed. Should be the same as uses on Form->control
+	 * @param array $googleOptions associative array of settings for places.Autocomplete
+	 *
+	 * @return string divContainer
+	 */
+	public function input( $fieldName, array $fieldOptions = [], $googleOptions = [] ) {
```

Comment:
Is this for BC?

Reply:
Yes, I would like to use the same methods as FormHelper
I can remove this or add @deprecated

Reply:
Adding the tag suffices :)